roles/contiv_network/defaults/main.yml

```diff
@@ -16,6 +16,8 @@ bgp_port: 179
 vxlan_port: 4789
 netplugin_rule_comment: "contiv network traffic"
 
+aci_gw_version: "contiv/aci-gw"
+aci_gw_image: "latest"
```

Comment:
@mapuri : Already added via this PR : https://github.com/contiv/demo/pull/97

But i think we need this default in ansible. aci-gw container is very much specific to ACI + Contiv and I dont think it will change often (Tightly depends on ACI releases). Also in net_demo_installer script I have a way to change these default variables via shell variables (just the way we did it for contiv_network_version). 

So If someone does not specifiy, it will take contiv/aci-gw:latest and if someone wants to change he can set these same shell variables and net_demo_installer script will take it and overwrite these defaults here.